src/webui/www/private/scripts/client.js

```diff
@@ -95,6 +95,61 @@ const getShowFiltersSidebar = function() {
     return (show === null) || (show === 'true');
 };
 
+// v4 and v6 js regex credit https://github.com/sindresorhus/ip-regex/blob/3e220cae3eb66ecfdf4f7678bea7306ceaa41c76/index.js#LL7C1-L25C56
+// license MIT
+const isValidIpAddress = (() => {
+    const v4 = '(?:25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]\\d|\\d)(?:\\.(?:25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]\\d|\\d)){3}';
+    const v6segment = '[a-fA-F\\d]{1,4}';
+
+    const v6 = `(?:(?:${v6segment}:){7}(?:${v6segment}|:)|                     // 1:2:3:4:5:6:7::  1:2:3:4:5:6:7:8
```

Comment:
Seems it still cannot detect `2001:db8:0:0:1::1` correctly.

Reply:
fixed